src/vtpty/LinuxPty.h

```diff
@@ -32,8 +32,8 @@ class LinuxPty final: public Pty
     class Slave final: public PtySlave
     {
       public:
-        int _slaveFd;
-        explicit Slave(PtySlaveHandle fd): _slaveFd { unbox<int>(fd) } {}
+        int slaveFd;
+        explicit Slave(PtySlaveHandle fd): slaveFd { unbox<int>(fd) } {}
```

Comment:
Actually, this looks more like `slaveFd` should never have been public, but private. So it should remain as `_slaveFd`, with leading underscore, because it's a private field member, but it should be made private in fact then, whereas the constructor is public of course.

Reply:
Changed it